watchdog/main.go

```diff
@@ -147,13 +147,10 @@ func main() {
 		return
 	}
 
-	readTimeout := time.Duration(config.readTimeout) * time.Second
```

Comment:
Good spot - this is actually duplicated now. But for testing you can do something like this:

```
$ cd faas/watchdog
$ go build
$ read_timeout=2 write_timeout=2 write_debug=true fprocess="sleep 8" ./watchdog
```

Then:

```
$ curl -v localhost:8080 -d "test"
```